diff --git a/node_modules/@testing-library/react-native/dist/preset/mock-modules.js b/node_modules/@testing-library/react-native/dist/preset/mock-modules.js
index 9356d33..9f82d25 100644
--- a/node_modules/@testing-library/react-native/dist/preset/mock-modules.js
+++ b/node_modules/@testing-library/react-native/dist/preset/mock-modules.js
@@ -35,10 +35,4 @@ jest.doMock('react-native/Libraries/Components/Picker/Picker', function () {
 
   return Picker;
 });
-jest.mock('react-native/Libraries/Animated/src/NativeAnimatedHelper').doMock('react-native/Libraries/Renderer/shims/ReactNative', function () {
-  var ReactNative = jest.requireActual('react-native/Libraries/Renderer/shims/ReactNative');
-  var NativeMethodsMixin = ReactNative.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.NativeMethodsMixin;
-  (0, _extends2.default)(NativeMethodsMixin, _mockNativeMethods.mockNativeMethods);
-  (0, _extends2.default)(ReactNative.NativeComponent.prototype, _mockNativeMethods.mockNativeMethods);
-  return ReactNative;
-});
\ No newline at end of file
+jest.mock('react-native/Libraries/Animated/src/NativeAnimatedHelper');
\ No newline at end of file
